app/components/guest_tag.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+import FormattedText from 'app/components/formatted_text';
+
+export default class GuestTag extends PureComponent {
+    static defaultProps = {
+        show: true,
```

Comment:
this default to false to prevent false/positives

Reply:
Ok, my reasoning here was, if I use `<GuestTag/>` directly it shows the tag, and is following the same pattern used by `<BotTag/>` component. Another option is to replace `show` with `hide` and set the default to `false`. Still a similar problem.